src/main/java/com/singlestore/jdbc/message/server/ColumnDefinitionPacket.java

```diff
@@ -139,30 +139,15 @@ public boolean isSigned() {
   public int getDisplaySize() {
     switch (dataType) {
       case VARCHAR:
-      case JSON:
```

Comment:
Could you let me know why you removed JSON from here?
Doesn't charset influence the length of JSON columns?